infra/modules/aws_github_environment_bootstrap/README.md

```diff
@@ -0,0 +1,218 @@
+# DX - AWS GitHub Environment Bootstrap
+
+![Terraform Module Downloads](https://img.shields.io/terraform/module/dm/pagopa-dx/aws-github-environment-bootstrap/aws?logo=terraform&label=downloads&cacheSeconds=5000&link=https%3A%2F%2Fregistry.terraform.io%2Fmodules%2Fpagopa-dx%2Faws-github-environment-bootstrap%2Faws%2Flatest)
+
+The AWS GitHub Environment Bootstrap module is designed for users who have just created a new GitHub repository and want to quickly focus on their goals without spending hours on setup. It is particularly useful for repositories that need to deploy to AWS infrastructure.
+
+The module performs the following actions:
+
+- Creates **AWS IAM Roles** for GitHub Actions to assume when deploying:
+  1. Infrastructure resources (IaC) - both CI (read-only) and CD (admin) roles.
+  2. Application resources - both CI (read-only) and CD (admin) roles.
+- Sets up **OpenID Connect (OIDC)** integration between GitHub Actions and AWS.
+- Creates **GitHub repository secrets** with the IAM role ARNs for workflows to use.
+- Uses the **pagopa-dx AWS provider** for consistent resource naming conventions.
+
+## Gotchas
+
+### Ensure Necessary AWS Permissions
+
+The AWS principal (user or IAM role) executing the `terraform apply` command must have sufficient permissions to:
+
+- Create and manage IAM roles and policies
+- Create OIDC identity providers
+- Attach policies to roles
+
+This typically requires `IAMFullAccess` or equivalent permissions.
+
+### GitHub Actions Workflow Configuration
+
+After deploying this module, configure your GitHub Actions workflows to assume the created IAM roles. Here's an example workflow configuration:
+
+```yaml
+name: Deploy Infrastructure
+on:
+  push:
+    branches: [main]
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    environment: infra-cd
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.INFRA_CD_ROLE_ARN }}
```

Comment:
Why `INFRA_CD_ROLE_ARN`? Should not be `ROLE_ARN` as it has already the environment boundaries?